core/services/fluxmonitorv2/poll_manager.go

```diff
@@ -4,13 +4,17 @@ import (
 	"time"
 
 	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/flux_aggregator_wrapper"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+const hibernationPollPeriod = 24 * time.Hour
+
 type PollManagerConfig struct {
 	IsHibernating      bool
```

Comment:
nit: this is a stateful variable not static config - maybe it belongs as a field on `PollManager` instead of here?